src/Compilers/VisualBasic/Portable/Syntax/Syntax.xml

```diff
@@ -5098,7 +5098,7 @@ Unify Enum declarations with other block type declarations.
         <description>The "As" keyword.</description>
       </child>
             
-      <child name="Type" kind="@TypeSyntax">
+      <child name="Type" optional="true" kind="@TypeSyntax">
         <description>The type-name part of the tuple element syntax.</description>
       </child>
```

Comment:
It looks like we should be using SimpleAsClauseSyntax here to group 'As' and 'type' together. This will be consistent with other similar places in the language (field/local declaration) and will prevent us from allowing trees where one piece is missing. Both the keyword and the type should be either present, or missing, but we shouldn't allow situations where only one of them is missing.